ui/components/app/toast-master/toast-master.js

```diff
@@ -256,6 +265,44 @@ function SwitchedNetworkToast() {
   );
 }
 
+function BaseNetworkToast() {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+
+  const showBaseNetworkToast = useSelector(
+    (state) => state.metamask.preferences.showBaseNetworkToast,
+  );
+
+  return (
+    showBaseNetworkToast && (
```

Comment:
Should we only show this if the network is the Base network? So that a user that is not on the Base network won't see it until they switch too the Base network? I guess this is a UX/Product question.